tests/test_users.py

```diff
@@ -67,3 +72,69 @@ def test_user_menu_devhub_links(base_url, selenium):
         # waiting for the homepage o reload
         page.wait_for_page_to_load()
         count += 1
+
+
+@pytest.mark.serial
+@pytest.mark.nondestructive
+def test_user_edit_profile(base_url, selenium, variables):
+    user = User(selenium, base_url).open().wait_for_page_to_load()
+    user.login('reusable_user')
+    # fill in the Edit profile form fields
+    user.edit.display_name(variables['display_name'])
+    user.edit.homepage_link(variables['homepage'])
+    user.edit.location(variables['location'])
+    user.edit.occupation(variables['occupation'])
+    user.edit.biography(variables['biography'])
+    user.edit.upload_picture()
+    # checks that the picture uploaded by the user is displayed
+    user.edit.profile_picture_is_displayed()
+    user.edit.submit_changes()
+    # checks that the updated display name is visible in the header
+    WebDriverWait(selenium, 10).until(
+        EC.text_to_be_present_in_element(
+            (
+                By.CSS_SELECTOR,
+                '.Header-user-and-external-links .DropdownMenu-button-text',
+            ),
+            variables['display_name'],
+        )
+    )
```

Comment:
Could you add this to the Base class? The Base class should contain anything that is always usable and I think the header is always displayed correct?

Reply:
Done!